### PR TITLE
[HCBS] QMS - FASI-1, FASI-2, HCBS-10 optional measures

### PR DESCRIPTION
**Pull request by rocio-desantiago**
### Description
 - MeasurePage tentatively updated to match changes from PR #80 
 - Some typos addressed
 - CMIT files reorganize to be consistent with order of required measures followed by optional measures similar to qm.ts file.
 - CMIT files updated to reflect measure details from this [excel sheet](https://docs.google.com/spreadsheets/d/1ztZ1DSYCUkKz9E_np5q-k9CgBoHYKEWUc_Fd2t3RJmc/edit?gid=1582150716#gid=1582150716)
    - Such as 
        | Measure | CMIT | Measure Specs | Delivery System | Data Source |
        | ------- | ------- | ------- | ------- | ------- |
        | FASI-1 | 969 | CMS | MTLSS, FFS | Record Review |
        | FASI-2 | 970| CMS | MTLSS, FFS | Record Review |
        | HCBS-10 | 111 | CMS | MTLSS | Case Management Records |
- Record Review added as a DataSource option
     - Added for the time being until we hear otherwise:
         Screenshot of comment on Data Source column in excel sheet of Measures

### Related ticket(s)
CMDCT-4155 (FASI-1)
CMDCT-4157 (FASI-2)
CMDCT-4159 (HCBS-10)

---
### How to test
Cloudfront URL: https://d1m8ad8ivtkepf.cloudfront.net/
1. Sign into HCBS, any state user
2. Create a QM report
3. On the sidebar menu, select [Optional Measure Results]. 
   - The 1st row - FASI-1, CMIT #969
   - The 2nd row - LTSS-5, CMIT #970
   - The 3rd row - LTSS-5, CMIT #111
4. Click [Edit] and you be taken to each respective measure overview page.
5. Check that the questions matches what is written in the jira tickets: [CMDCT-4155](https://jiraent.cms.gov/browse/CMDCT-4155) , [CMDCT-4155](https://jiraent.cms.gov/browse/CMDCT-4155) , [CMDCT-4159](https://jiraent.cms.gov/browse/CMDCT-4159) 
6. For FASI-1, you will see this page
FASI-1 Measure Overview Page
8. For FASI-2, you will see this page
FASI-2 Measure Overview Page
9. For HCBS-10, you will see this page
HCBS-10Measure Overview Page

---
### Author checklist

- [x] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Comment by rocio-desantiago** [comment]
This is the code climate thing that's complaining:
Codeclimate error encouraging a refactor of qm.ts file

**Review by ailZhou — DISMISSED** [review]
lgtm! tested on deploy and all the pages are showing up and matching the ticket's text

File: services/app-api/forms/qm.ts [hunk 1]
```diff
@@ -687,22 +687,232 @@ export const qmReportTemplate: ReportTemplate = {
...
+                    "What is the name of the agency or entity that audited or validated the report?",
+                },
+              ],
+            },
+          ],
+        },
+        {
+          type: ElementType.Radio,
+          label:
+            "Did you deviate from the [reportYear] Technical Specifications?",
```
Comment (benmartin-coforma) [inline services/app-api/forms/qm.ts:731]: I saw in PR #83 we changed "deviate/deviation" to "follow/variance" - should we do the same here right off the bat?

Same comment for lines 811 and 892.

**Review by benmartin-coforma — APPROVED** [review]
I just have one question, but beyond that I approve!

I definitely disagree with CodeClimate on the max length of file; for our purposes it would make way less sense to break that code up.

File: services/app-api/forms/qm.ts (hunk 1 above)
Comment (rocio-desantiago) [reply services/app-api/forms/qm.ts]: omg How I missed that. Thank you!

**Review by benmartin-coforma — APPROVED** [review]

**Review by ailZhou — APPROVED** [review]

**Pull request merged**